.circleci/config.yml

```diff
@@ -42,6 +42,30 @@ jobs:
       - run: helm package ./charts/atlantis
       - run: |
           aws s3 cp atlantis-*.tgz s3://procore-helm-charts/
+  build_docker:
+    docker:
+      - image: docker:latest
+    working_directory: ~/atlantis-helm-chart
+    steps:
+      - setup_remote_docker
+      - checkout
+      - run:
+          name: Build Image
+          command: |
+            docker build \
+              --pull \
+              -t quay.io/procoredevops/atlantis-helm-chart:$CIRCLE_TAG ~/atlantis-helm-chart/procore-atlantis/
```

Comment:
Do you have to do any cleanup after we build?

Reply:
Nope. The build is all done in a remote docker environment. The built image is cleaned up automatically.